modules/trainer/GenericTrainer.py

```diff
@@ -606,14 +671,14 @@ def sample_commands_fun():
                         self.model, self.config, lr_scheduler, self.tensorboard
                     )
 
-                    self.tensorboard.add_scalar("loss/loss", accumulated_loss, train_progress.global_step)
+                    self.tensorboard.add_scalar("loss/train_step", accumulated_loss, train_progress.global_step)
```

Comment:
Additional thought: generally, validation loss is calculated per epoch. So we should consider listing the training loss per epoch as well. Or, since how often you run validation loss is configurable here, we should list the training loss per the same time period as well.

Reply:
i think listing train loss and validation loss per global step is okay, real period of calculation doesn't matter as tensorboard interpolate graph between dots